aspnet/hosting/apppool.rst

```diff
@@ -0,0 +1,42 @@
+.. _apppool:
```

Comment:
This explains how to change app pool identities but not why. You need to bring in the App Pool Identity Accounts section from https://www.iis.net/learn/manage/configuring-security/application-pool-identities